src/game/systems/Collision.cpp

```diff
@@ -65,8 +65,8 @@ namespace game::systems
                         else if (player && player->inventory[Item::Identifier::KickShoes]) {
                             raylib::core::Vector3f posDelta = pos2 - pos1;
                             /// Kick bomb if the player is moving toward it.
-                            if ((abs(posDelta.x) > abs(posDelta.z)) ? (posDelta.x * vel1.x > 0)
-                                                                    : (posDelta.z * vel1.z > 0))
+                            if ((fabsf(posDelta.x) > fabsf(posDelta.z)) ? (posDelta.x * vel1.x > 0)
```

Comment:
You are missing the `<cmath>` include, also use `std::fabsf` to avoid calling a random macro